src/MSAL.PCL/AbstractClientApplication.cs

```diff
@@ -209,13 +209,13 @@ protected AbstractClientApplication(string authority, string clientId, string re
                 parameters = PlatformPlugin.DefaultPlatformParameters;
             }
 
-            var handler = new AcquireTokenSilentHandler(this.GetHandlerData(authenticator, scope, policy, this.UserTokenCache), user, parameters, forceRefresh);
+            var handler = new SilentRequest(this.GetHandlerData(authenticator, scope, policy, this.UserTokenCache), user, parameters, forceRefresh);
             return await handler.RunAsync().ConfigureAwait(false);
         }
 
-        internal virtual HandlerData GetHandlerData(Authenticator authenticator, string[] scope, string policy, TokenCache cache)
+        internal virtual RequestData GetHandlerData(Authenticator authenticator, string[] scope, string policy, TokenCache cache)
```

Comment:
Check the latest architecture doc, I think we change it to AuthenticationRequestParameters